semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -250,25 +250,25 @@ def parse_id(attribute):
             position = attribute.lc.data[list(attribute)[0]]
             msg = "Non array examples for {} are not allowed".format(attr_type)
             raise ValidationError.from_yaml_pos(position, msg)
-        if (
-            (is_simple_type or isinstance(attr_type, EnumAttributeType))
-            and not isinstance(examples, CommentedSeq)
-            and examples is not None
-        ):
+        if not isinstance(examples, CommentedSeq) and examples is not None:
+            # TODO: If validation fails later, this will crash when trying to access positio data
```

Comment:
I'm under the impression that someone needs to read the YAML library documentation in more depth and refactor a lot of code (but not in this PR).